packages/unplugin-vue-i18n/src/index.ts

```diff
@@ -703,9 +703,11 @@ async function generateBundleResources(
     }
   }
 
-  return `export default {
-  ${codes.join(`,\n`)}
-}`
+  return `import merge from 'lodash.merge'
```

Comment:
this code will add `lodash.merge' as dependecy. It means that the code of `lodash.merge'` will be bundled.
As unplulgin-vue-i18n, we must avoid adding dependencies and affecting the code size of the application.
If merge is to be supported, it must have its own merge handling.

Reply:
I replaced `lodash.merge` with a deep merge solution from https://stackoverflow.com/a/34749873.